recipes/kakoune/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -exo pipefail
+
+if [[ "$OSTYPE" == "linux-gnu"* ]]; then
+    make install -j${CPU_COUNT} \
+        debug=no \
+        PREFIX=${PREFIX} \
+        CXX=${CXX}
+
+elif [[ "$OSTYPE" == "darwin"* ]]; then
```

Comment:
```suggestion
elif [[ "${target_platform}" == "darwin"* ]]; then
```

Reply:
For MacOS, `target_platform` value is not `darwin` but `osx-*`.